server/enterprise/message_export/message_export_test_e2e_generators.go

```diff
@@ -110,10 +208,154 @@ type SetupReturn struct {
 	channels          []*model.Channel
 	teams             []*model.Team
 	batchTimes        []batchStartEndTimes
+	jobExportDir      string
+}
+
+func generateActianceBatchTest1(t *testing.T, th *api4.TestHelper, attachmentDir, exportDir string,
+	attachmentBackend filestore.FileBackend) JobResults {
+	waitUntilZeroPosts(t, th)
+	now := model.GetMillis()
+	jobStart := now - 1
+
+	th.App.UpdateConfig(func(cfg *model.Config) {
+		*cfg.MessageExportSettings.EnableExport = true
+		*cfg.MessageExportSettings.ExportFromTimestamp = jobStart
+		*cfg.MessageExportSettings.BatchSize = 5
+		*cfg.MessageExportSettings.ExportFormat = model.ComplianceExportTypeActiance
+		*cfg.FileSettings.DriverName = model.ImageDriverLocal
+		*cfg.FileSettings.Directory = attachmentDir
+
+		if exportDir != attachmentDir {
+			*cfg.FileSettings.DedicatedExportStore = true
+			*cfg.FileSettings.ExportDriverName = model.ImageDriverLocal
+			*cfg.FileSettings.ExportDirectory = exportDir
+		}
+	})
+
+	contents := []string{"Hello there"}
+	attachmentPath001 := "path/to/attachments/one.txt"
+	_, _ = attachmentBackend.WriteFile(bytes.NewBufferString(contents[0]), attachmentPath001)
+	post, err := th.App.Srv().Store().Post().Save(th.Context, &model.Post{
+		ChannelId: th.BasicChannel.Id,
+		UserId:    model.NewId(),
+		Message:   "zz" + model.NewId() + "b",
```

Comment:
What's the rationale for generating these ids? Is it something that could be refactored (e.g. similarly to the `NewTestId` helper we have in app).

Reply:
I have no idea, that's how it was done prior to me, and I just kept it to avoid breaking things. I'll see if I can simplify now.

Reply:
Oh yeah, `NewTestId` is meant to fix these hacks. Please feel free to use that.